addons/spreadsheet/static/tests/pivots/model/pivot_plugin_test.js

```diff
@@ -647,6 +648,65 @@ QUnit.module("spreadsheet > pivot plugin", {}, () => {
         assert.strictEqual(getEvaluatedCell(model, "B1").format, "dd/mm/yyyy");
     });
 
+    QUnit.test("PIVOT.TABLE formulas results", async function (assert) {
+        const { model } = await createSpreadsheetWithPivot({
+            arch: /* xml */ `
+                <pivot>
+                    <field name="product_id" type="col"/>
+                    <field name="foo" type="row"/>
+                    <field name="id" type="row"/>
+                    <field name="probability" type="measure"/>
+                </pivot>`,
+        });
+        const sheetId = model.getters.getActiveSheetId();
+        model.dispatch("CREATE_SHEET", { sheetId: "42" });
+        setCellContent(model, "A1", `=ODOO.PIVOT.TABLE("1")`, "42");
+        for (let col = 0; col <= 3; col++) {
```

Comment:
replace 3 and 10 with the dimensions of the table, seems quite arbitrary. Also might be worth making sure that we don't spread beyond the range. I think you introduced `checkFunctionDoesntSpreadBeyondRange` in o-spreadsheet to that end?